src/write_sensu.c

```diff
@@ -115,9 +115,7 @@ static int sensu_connect(struct sensu_host *host) /* {{{ */
 		host->res = NULL;
 		hints.ai_family = AF_INET;
 		hints.ai_socktype = SOCK_STREAM;
-#ifdef AI_ADDRCONFIG
-		hints.ai_flags |= AI_ADDRCONFIG;
-#endif
+		hints.ai_flags = AI_ADDRCONFIG;
```

Comment:
Looking at the context, it looks like `hints` is only used within the `if (!READY)` block. Can we move it into the if block?